### DISCUSSION TITLE
Config: Check a rule but don't fix it

### DISCUSSION THREAD

**Discussion by schulzjo-tng** [cqfn/diKTat#1921] [Q&A]
I want `diktatCheck` and `diktatFix` to _test_ the rule `VARIABLE_HAS_PREFIX` but I don't want `diktatFix` to _change_ anything. The reason being that I have some data classes that model an API response violating this rule. I am aware that I can `@Suppress` this rule per class or per file, but I sometimes don't notice when `diktatFix` changes something. Can this be configured?

**Comment by orchestr7** [comment]
hey! `@Suppress` can be applied to a particular statement, including a single variable. If it is applied to the statement, this statement will never be fixed or checked

**Reply by schulzjo-tng** [reply]
Thank you for the response. I am aware that this is possible when I encounter this issue on a specific class. I am still wondering if I can configure the rule project-wide. For example, let's say I need to map a new API response that has a rule-violating value. I don't notice it immediately and hence I forget to put an `@Suppress` on that class. Later I routinely execute `diktatFix` before I commit the new file. Then I don't notice that the semantics of the file have been changed until the code breaks in tests or in production.
